test/extended/util/disruption/disruption.go

```diff
@@ -84,35 +85,29 @@ type TestData struct {
 // test is being executed. Description is used to populate the JUnit suite name, and testname is
 // used to define the overall test that will be run.
 func Run(f *framework.Framework, description, testname string, adapter TestData, invariants []upgrades.Test, fn func()) {
-	testSuite := &junit.TestSuite{Name: description, Package: testname}
-	test := &junit.TestCase{Name: testname, Classname: testname}
-	testSuite.TestCases = append(testSuite.TestCases, test)
+	testSuite := &junitapi.JUnitTestSuite{Name: description}
```

Comment:
Package is not present under our normal test suite XML, so I dropped it. I did keep using it for the filename however so that stays consistent in case anyone is looking for it by a specific name. Note description="Cluster upgrade", and package="upgrade" here.